webpack.config.js

```diff
@@ -111,23 +111,13 @@ function getWebpackConfig( { extensionName = '' } = {}, argv ) {
 			chunkIds: isDevelopment ? 'named' : 'natural',
 			minimize: shouldMinify,
 			minimizer: [
-				new UglifyJsPlugin( {
+				new TerserPlugin( {
 					cache: 'docker' !== process.env.CONTAINER,
 					parallel: true,
 					sourceMap: Boolean( process.env.SOURCEMAP ),
-					uglifyOptions: {
-						compress: {
-							/**
-							 * Produces inconsistent results
-							 * Enable when the following is resolved:
-							 * https://github.com/mishoo/UglifyJS2/issues/3010
-							 */
-							collapse_vars: false,
-						},
-						mangle: {
-							safari10: true,
-						},
+					terserOptions: {
 						ecma: 5,
```

Comment:
`ecma` options are available for parse, compress and output. Would it make sense to keep all of these at the highest `ecma` version available, and let babel deal with output?

https://github.com/fabiosantoscode/terser#minify-options